Framework/Geometry/test/ComponentInfoTest.h

```diff
@@ -143,7 +144,7 @@ class ComponentInfoTest : public CxxTest::TestSuite {
     auto scaleFactors = boost::make_shared<std::vector<Eigen::Vector3d>>(2);
     auto internalInfo = Kernel::make_unique<Beamline::ComponentInfo>(
         detectorIndices, detectorRanges, componentIndices, componentRanges,
-        parentIndices, positions, rotations, scaleFactors, -1, -1);
+        parentIndices, isVisible, positions, rotations, scaleFactors, -1, -1);
     Mantid::Geometry::ObjComponent comp1("component1");
     Mantid::Geometry::ObjComponent comp2("component2");
 
```

Comment:
Tests fixed, but not extended in anyway. I would argue that putting the tests on `Beamline::ComponentInfo` is a minimum. Since `Geometry::ComponentInfo` class is mainly pass-through, not quite as important.